jenkins/Jenkinsfile

```diff
@@ -19,68 +19,79 @@ pipeline {
         }
         stage('Test') {
             parallel {
-                stage('unit') {
-                    // ensures that this stage will get assigned its own workspace
-                    agent { label 'candlepin' }
-                    steps {
-                        sh 'sudo chown -R jenkins:jenkins $WORKSPACE'
-                        checkout scm
-                        sh 'sh jenkins/unit-tests.sh'
-                    }
-                    post {
-                        always {
-                            sh 'sh jenkins/cleanup.sh'
-                            archiveArtifacts artifacts: 'unit-artifacts/**',
-                                             fingerprint: true,
-                                             onlyIfSuccessful: false
```

Comment:
Why was this removed? Without it, the unit test logs no longer appear in the "Artifacts" tab in jenkins

Reply:
Because those artifacts are collected from docker and so are only relevant for spec test runs. For unit tests i replaced this with junit report collector. Check tests panel next to artifacts in Jenkins.

Reply:
Ok, that looks good. I assume any failing tests will be shown up first, to avoid having to scroll through thousands of tests and click 'Show More' at the bottom of the page hundreds of times?

Reply:
Maybe you can purposely introduce a broken unit test to see if it works like that?

Reply:
Done.